app/services/hyrax/collection_types/create_service_decorator.rb

```diff
@@ -209,7 +118,7 @@ def self.create_user_collection_type
       #   If calling from Abilities, pass the ability.
       #   If you try to get the ability from the user, you end up in an infinite loop.
       # rubocop:disable Metrics/MethodLength
-      def self.add_default_participants(collection_type_id)
+      def add_default_participants(collection_type_id)
```

Comment:
I'd feel better if this were in a submodule `ClassMethods`

Reading this module it would be easy to think it's an instance method.  But we prepend it to a singleton class.